.goreleaser.yml

```diff
@@ -16,37 +16,50 @@ builds:
     - arm64
     - ppc64le
     - s390x
+  - id: unpack
+    main: ./cmd/unpack/main.go  # Update this path to the actual location of your unpack source
+    binary: bin/unpack
+    asmflags: "{{ .Env.GO_BUILD_ASMFLAGS }}"
+    gcflags: "{{ .Env.GO_BUILD_GCFLAGS }}"
+    ldflags: "{{ .Env.GO_BUILD_LDFLAGS }}"
+    goos:
+      - linux
+    goarch:
+      - amd64
+      - arm64
+      - ppc64le
+      - s390x
 dockers:
 - image_templates:
   - "{{ .Env.IMAGE_REPO }}:{{ .Env.IMAGE_TAG }}-amd64"
   dockerfile: Dockerfile
   goos: linux
   goarch: amd64
-  use: buildx
+  use: docker
```

Comment:
Why are we switching from `buildx` to `docker` here?

Reply:
For reference, goreleaser's documentation states that there are 2 ways to do multi-platform images, using `ko` or `buildx`: https://goreleaser.com/cookbooks/multi-platform-docker-images/

IIRC we set the use of `buildx` so we could publish multi-platform images by creating these separate platform manifests and publishing them under a single tag

Reply:
I _think_ this was done in order to get `alias docker=podman` to make it possible to run `make release` with just podman installed and have it work. But if that breaks multi-arch builds, I agree that we have to change it back.

Bryce, here's your original PR that made multi-arch work: https://github.com/operator-framework/operator-controller/pull/186

Reply:
It was done to allow for _local_ alias of podman and I think it got inadvertently committed. Good catch. Reverted it in 2402bb67